### DISCUSSION TITLE
I have 2 features required

### DISCUSSION THREAD

**Discussion by mybrosavages** [Zrips/CMI#4450] [Feature Requests]
First is custom craft can add the permission per craft and the crafting can saving the amount of the recipes
Second I would like to have to servermeta the global value work same as usermeta but it’s for the server for example I can set the placeholder change with schedule with this servermeta or I can apply it with usermeta + - number so I can making the sale item event time or something like that and I can make the vote for event that I’m making not the vote for player. Thanks CMI best developers

**Comment by mybrosavages** [comment]
Little bit more the servermeta are the all player value when for example if I do like /servermeta add test test
When I used %cmi_user_meta_test% all the player will see it as test and it’s good for some custom command for player that never join the server.

**Comment by thboehi** [comment]
I like the second idea about the server meta, global players metadata.
